app/components/claim-user.js

```diff
@@ -0,0 +1,90 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { alias } from '@ember/object/computed';
+import Analytics from 'ember-osf/mixins/analytics';
+import { task } from 'ember-concurrency';
+import config from 'ember-get-config';
+import $ from 'jquery';
+import { validator, buildValidations }
+    from 'ember-cp-validations';
+
+const Validations = buildValidations({
+    email: {
+        validators: [
+            validator('presence', true),
+            validator('format', {
+                type: 'email',
+                regex: /^[-a-z0-9~!$%^&*_=+}{'?]+(\.[-a-z0-9~!$%^&*_=+}{'?]+)*@([a-z0-9_][-a-z0-9_]*(\.[-a-z0-9_]+)*\.(aero|arpa|biz|com|coop|edu|gov|info|int|mil|museum|name|net|org|pro|travel|mobi|[a-z][a-z])|([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}))(:[0-9]{1,5})?$/i,
+            }),
+        ],
+    },
+});
+
+/**
+ * @module ember-preprints
+ * @submodule components
+ */
+
+/**
+ * Confirm share preprint modal
```

Comment:
er this was all copy pasta  I didn't rewrite, can you make it accurate for this new component